src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -207,6 +209,24 @@ public StratisMain()
 
             Assert(this.Consensus.HashGenesisBlock == uint256.Parse("0x0000066e91e46e5a264d42c89e1204963b2ee6be230b443e9159020539d972af"));
             Assert(this.Genesis.Header.HashMerkleRoot == uint256.Parse("0x65a26bc20b0351aebf05829daefa8f7db2f800623439f3c114257c91447f1518"));
+
+            this.RegisterMempoolRules(this.Consensus);
+        }
+
+        protected void RegisterMempoolRules(IConsensus consensus)
+        {
+            consensus.MempoolRules = new List<Type>()
```

Comment:
Later we can star to separate rules (I think there are rules that only apply to POS)

Reply:
Agreed